README.md

```diff
@@ -1,2 +1,45 @@
 # ambari-presto-service
-Ambari service for Presto
+
+**Note: the integration code in this repo is still a work in progress. It may be lacking certain integration features and at times may break. Use at your own risk.**
+
+This repo contains code and configuration needed to integrate [Presto](https://prestodb.io/) with [Ambari](https://ambari.apache.org/). Once you've deployed the integration code (for more details on this see [Integration instructions for users](#iifu)), you'll be able to do the following:
+
+1. Install and deploy Presto to your cluster from the Ambari UI.
+2. Configure all aspects of Presto via the Ambari UI.
+3. Have dashboard access to some critical Presto and kernel level metrics.
+
+## Requirements
+
+1. Python 2.6/2.7.
+2. Make.
+
+## <a name="iifu"></a>Integration instructions for users
+
+Unfortunately, at the moment Ambari does not support a more user friendly installation method. The installation has to be done by following the somewhat manual steps outlined below.
+
+1. Create the distribution by executing `make clean dist`. This will create a `tar.gz` in the dist directory. This file is the distribution you'll need to upload to your cluster.
+2. Upload the distribution archive to the node in your cluster that runs the Ambari master server.
+3. Assuming that `ambari-server` is installed in `/var/lib/ambari-server`, you should execute the following:
+```
+mkdir /var/lib/ambari-server/resources/stacks/HDP/2.2/services/PRESTO
+tar -xvf ambari-presto-0.1.0.linux-x86_64.tar.gz -C /var/lib/ambari-server/resources/stacks/HDP/2.2/services/PRESTO
```

Comment:
Yes, I'm impressed you caught that :) That's why by default in config.properties the port for Presto is set to 8081.